plugins/org.jkiss.dbeaver.net.ssh.ui/src/org/jkiss/dbeaver/ui/net/ssh/SSHTunnelConfiguratorUI.java

```diff
@@ -235,26 +234,28 @@ private static void setNumberEditStyles(Text text) {
     }
 
     @Nullable
-    private static DBPAuthInfo promptCredentialsDialog(@NotNull SSHConstants.AuthType type,
-        @NotNull DBWHandlerConfiguration configuration) {
-        final DBPAuthInfo credentialsToConnectTo;
+    private static DBPAuthInfo promptCredentialsDialog(
+        @NotNull SSHConstants.AuthType type,
+        @NotNull DBWHandlerConfiguration configuration
+    ) {
         try {
-            credentialsToConnectTo = DBWorkbench.getPlatformUI()
-                .promptUserCredentials(SSHUIMessages.model_ssh_dialog_credentials,
+            return DBWorkbench.getPlatformUI().promptUserCredentials(
+                    SSHUIMessages.model_ssh_dialog_credentials,
                     SSHUIMessages.model_ssh_dialog_credentials_username,
                     configuration.getUserName(),
                     type.equals(SSHConstants.AuthType.PUBLIC_KEY)
-                    ? SSHUIMessages.model_ssh_dialog_credentials_passphrase
-                    : SSHUIMessages.model_ssh_dialog_credentials_password,
+                        ? SSHUIMessages.model_ssh_dialog_credentials_passphrase
+                        : SSHUIMessages.model_ssh_dialog_credentials_password,
                     configuration.getPassword(),
                     type.equals(SSHConstants.AuthType.PUBLIC_KEY),
                     false
                 );
         } catch (Exception e) {
-            log.error(e);
+            DBWorkbench.getPlatformUI().showError(
+                CoreMessages.dialog_connection_wizard_start_dialog_error_title,
+                e.getMessage());
```

Comment:
Please pass the exception itself
```suggestion
            DBWorkbench.getPlatformUI().showError(
                CoreMessages.dialog_connection_wizard_start_dialog_error_title,
                e.getMessage().
                e
            );
```